src/api/app/views/webui/packages/job_history/index.html.haml

```diff
@@ -31,7 +31,7 @@
           = time_tag(Time.at(jobhistory.ready_time))
         %td
           - if jobhistory.reason == 'source change'
-            = link_to(jobhistory.reason, package_rdiff_path(project: @project.name, package: @package.name, orev: jobhistory.prev_srcmd5, rev: jobhistory.srcmd5))
+            = link_to(jobhistory.reason, package_rdiff_path(project: @project.name, package: @package, orev: jobhistory.prev_srcmd5, rev: jobhistory.srcmd5))
```

Comment:
it is not really related with the changes.. better in a different commit :thinking: